client/java/src/main/java/io/openlineage/client/transports/CompositeConfig.java

```diff
@@ -28,13 +28,16 @@ public final class CompositeConfig implements TransportConfig, MergeConfig<Compo
 
   @Getter @Setter private List<TransportConfig> transports;
 
-  @Getter @Setter private boolean continueOnFailure;
+  @Getter @Setter private Boolean continueOnFailure;
+
+  @Getter @Setter private Boolean threadPoolActive;
```

Comment:
IMHO this naming is a bit confusing. What do you think about `useThreadPool` or `withThreadPool`?